plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultJdbcMetadata.java

```diff
@@ -138,7 +138,7 @@ public void testGetColumnHandles()
         assertEquals(metadata.getColumnHandles(SESSION, tableHandle), ImmutableMap.of(
                 "text", new JdbcColumnHandle("TEXT", JDBC_VARCHAR, VARCHAR),
                 "text_short", new JdbcColumnHandle("TEXT_SHORT", JDBC_VARCHAR, createVarcharType(32)),
-                "value", new JdbcColumnHandle("VALUE", JDBC_BIGINT, BIGINT)));
+                "value", new JdbcColumnHandle("value", JDBC_BIGINT, BIGINT)));
```

Comment:
Why does just changing the case works? Do you mean H2 considers "VALUE" as reserved but "value" is not reserved?

If this is the case then also mention this (and fix the case) in the commit message.

Reply:
Also why not change `value` to `val` everywhere with H2?

Reply:
I've quoted this column. I can make it val everywhere instead

Reply:
I think val is easier than worrying about quoting. On the other hands it seems these tests are read-only so maybe quoting is not too painful? I don't have strong opinion. I'll defer to your judgement.